win32ss/user/ntuser/winpos.c

```diff
@@ -2301,6 +2285,27 @@ co_WinPosSetWindowPos(
       WinPos.flags |= SWP_NOZORDER|SWP_NOREDRAW;
    }
 
+   if (VisBefore != NULL)
```

Comment:
(Note: the `!= NULL` is optional)

Reply:
I'm fully aware, but I prefer to spell it out, mostly to re-affirm to the reader that it's a pointer data type.